pkg/util/system/network_linux.go

```diff
@@ -68,6 +68,96 @@ func ParseProcessRoutes(procPath string, pid int) ([]NetworkRoute, error) {
 	return routes, nil
 }
 
+// ParseProcessIPs parses /proc/<pid>/net/fib_trie and returns the /32 IP
+// addresses found. The result does not contain duplicate IPs.
+//
+// Here's an example of /proc/<pid>/net/fib_trie that shows its format:
+//Main:
```

Comment:
I think you'll want to indent this by a tab, otherwise it will be wrapped as plain text by godoc.  Alternately, if the kernel documents this format then it may make sense to just link to that doc.

Reply:
Thanks. I fixed the indentation in a new commit :+1: